examples/b2c/checkout/integrations/classes/B2CDeliverySample.cls

```diff
@@ -241,19 +280,35 @@ global class B2CDeliverySample implements sfdc_checkout.CartShippingCharges {
         // ReferenceNumber: Reference Number from External Service
         // IsActive: If this Option is Active
 
+        // Below fields are available only for api version >= 61
+
+        // TransitTimeMin: Minimum Transit Time
+        // TransitTimeMax: Maximum Transit Time
+        // TransitTimeUnit: Time Unit for Transit Time (Valid Values are "Hours", "Days", "Weeks")
+        // ProcessTime: Process Time
+        // ProcessTimeUnit: Time Unit for Process Time (Valid Values are "Hours", "Days", "Weeks")
+
         Id productId = getDefaultShippingChargeProduct2Id();
-        CartDeliveryGroupMethod cartDeliveryGroupMethod = new CartDeliveryGroupMethod(
-            CartDeliveryGroupId = cartDeliveryGroupId,
-            ExternalProvider = shippingOption.getProvider(),
-            Name = shippingOption.getName(),
-            ShippingFee = shippingOption.getRate(),
-            WebCartId = webCartId,
-            Carrier = shippingOption.getCarrier(),
-            ClassOfService = shippingOption.getClassOfService(),
-            ProductId = productId,
-            ReferenceNumber = shippingOption.getReferenceNumber(),
-            IsActive = shippingOption.isActive()
-        );
+
+        CartDeliveryGroupMethod cartDeliveryGroupMethod = new CartDeliveryGroupMethod();
```

Comment:
This is CartDeliveryGroupMethodDto, correct?

Reply:
@wei-liu-salesforce  We do not use DTO in legacy Apex Integration Samples.